app/src/lib/markdown-filters/team-mention-filter.ts

```diff
@@ -90,7 +95,8 @@ export class TeamMentionFilter implements INodeFilter {
         org === undefined ||
         team === undefined ||
         // Team references are only added when the repository owner is the org to prevent linking to a team outside the repositories org.
-        org.slice(1) !== this.repository.owner.login
+        org.slice(1).toLocaleLowerCase() !==
+          this.repository.owner.login.toLocaleLowerCase()
```

Comment:
we should only use locale-aware string methods when dealing with data that is to be presented to the user (and currently not even that given that we don't localize the app) such as for sorting.

While using `localeLowerCase` _on both sides_ of a comparison like this is _probably_ safe there's no reason for us to bring locales into this and it can have unintended consequences. The typical example is Turkish where the lower case of `I` isn't `i`, but `ı`. Localization is super hard as I've once again learned this week and we don't want application logic to change (potentially) depending on a user's locale.

```suggestion
        caseInsensitiveEquals(org.slice(1), this.repository.owner.login)
```